### PR TITLE
Play 77

### PR DESCRIPTION
**Pull request by captainhurst**
https://strongmind.atlassian.net/browse/PLAY-77

### CODE REVIEW

File: app/jsx/bundles/common.js [hunk 1]
```diff
@@ -117,6 +118,19 @@ if (
...
 }
 
+if(!window.ENV.IS_STUDENT){
+  $(document).ready(function(){
+    let showTeacherTools = $("#sm-teacher-tools").find("a").length
+    if (showTeacherTools === 0 ){
+      $(".sm-teacher-tools-container").hide();
+    }
+    let activeDropdown = $("#sm-teacher-tools").find("a").hasClass("active")
+    console.log("activeDropdown", activeDropdown);
```
Comment (markng) [inline app/jsx/bundles/common.js]: remove debug statements

File: app/presenters/course_for_menu_presenter.rb [hunk 2]
```diff
@@ -31,6 +31,7 @@ def initialize(course, available_section_tabs, user = nil, context = nil)
     @available_section_tabs = (available_section_tabs || []).select do |tab|
       DASHBOARD_CARD_TABS.include?(tab[:id])
     end
+
```
Comment (markng) [inline app/presenters/course_for_menu_presenter.rb]: don't care that much, but this is a solo change in a file that doesn't change anything :)

File: app/stylesheets/base/_ic_app_layout.scss [hunk 3]
```diff
@@ -187,7 +191,7 @@ body:not(.course-menu-expanded) {
   }
 
   body.course-menu-expanded:not(.ic-no-flex-layout):not(.embedded):not(.is-inside-submission-frame) & {
-    margin-left: $ic-left-side-width;
+    // margin-left: $ic-left-side-width;
```
Comment (markng) [inline app/stylesheets/base/_ic_app_layout.scss]: remove commented rule

File: app/stylesheets/base/_ic_app_layout.scss [hunk 4]
```diff
@@ -249,12 +249,124 @@ body:not(.course-menu-expanded) {
   }
 }
 
+.sm-course-menu.list-view{
+  padding: 0 10px;
+  position: relative;
+  border-bottom: $sm-gray-4;
```
Comment (markng) [inline app/stylesheets/base/_ic_app_layout.scss]: I'm quite nervous (though not necessarily a complete block) on using variable names that reflect an actual style rather than a purpose. I'm particularly afraid that this will become a pattern (though I understand your need for a temporary fix).

File: app/stylesheets/base/_ic_app_layout.scss [hunk 5]
```diff
@@ -249,12 +249,124 @@ body:not(.course-menu-expanded) {
...
+
+  ul{
+    li{
+      float: left;
+
+      a{
+        border-radius: 0px;
+        padding: 20px 15px;
+        // margin-right: 20px;
```
Comment (markng) [inline app/stylesheets/base/_ic_app_layout.scss]: commented css rule.

File: app/views/courses/settings.html.erb [hunk 6]
```diff
@@ -485,6 +485,7 @@
       <% @context.course_sections.active.order(CourseSection.best_unicode_collation_key('name')).each do |section| %>
         <li class="section">
           <% if can_do section, @current_user, :read %>
+            Farts are funny
```
Comment (markng) [inline app/views/courses/settings.html.erb]: yes. They are.

File: yarn.lock [hunk 7]
```diff
@@ -3017,8 +3017,8 @@ fbjs@^0.6.1:
     whatwg-fetch "^0.9.0"
 
 fbjs@^0.8.8, fbjs@^0.8.9:
-  version "0.8.15"
-  resolved "https://registry.yarnpkg.com/fbjs/-/fbjs-0.8.15.tgz#4f0695fdfcc16c37c0b07facec8cb4c4091685b9"
+  version "0.8.16"
+  resolved "https://registry.yarnpkg.com/fbjs/-/fbjs-0.8.16.tgz#5e67432f550dc41b572bf55847b8aca64e5337db"
```
Comment (markng) [inline yarn.lock]: Do we know what this version change does? Why is it required?

File: yarn.lock [hunk 8]
```diff
@@ -7378,9 +7370,9 @@ script-loader@^0.7.0:
   dependencies:
     raw-loader "~0.5.1"
 
-"scroll-into-view@git+https://github.com/bkirkby/scroll-into-view.git#588b0ced98eeecf84e6fb2074aa076e80b7cffab":
+"scroll-into-view@https://github.com/bkirkby/scroll-into-view.git#588b0ced98eeecf84e6fb2074aa076e80b7cffab":
   version "1.7.4"
-  resolved "git+https://github.com/bkirkby/scroll-into-view.git#588b0ced98eeecf84e6fb2074aa076e80b7cffab"
+  resolved "https://github.com/bkirkby/scroll-into-view.git#588b0ced98eeecf84e6fb2074aa076e80b7cffab"
```
Comment (markng) [inline yarn.lock]: I assume these are the same thing? Not sure why this change happened.

File: yarn.lock (hunk 7 above)
Comment (captainhurst) [reply yarn.lock]: No, that was autogenerated. I'll bet that version updated because some dependency isn't pegged to a patch version.

File: yarn.lock (hunk 8 above)
Comment (captainhurst) [reply yarn.lock]: I'd assume the update was because we're pegged to a minor version and not the patch version

**Comment by yuric** [comment]
https://github.com/instructure/canvas-lms/commit/5c3abaefe8233e7f331ee090f2a6e8db3021db45#diff-d49a3bb2a866ad36525d52caa8075aa6

=(
